src/patternfly/components/Button/examples/Button.md

```diff
@@ -94,6 +130,23 @@ import './Button.css'
 {{#> button button--modifier="pf-m-control" button--attribute="disabled"}}
   Control disabled
 {{/button}}
+<br>
```

Comment:
did you mean to add this section to the disabled buttons?

Reply:
yes i was assuming the small text would apply to all of the buttons with text. @mceledonia what do you think?

Reply:
They're in the disabled section, but aren't disabled. You already have examples of them in the "variations" example to show what they look like. I don't know that they need to be listed under disabled, even if we update them to be disabled.

Reply:
Oh yep my bad don't know how they got in there